backend/dataall/modules/datasets/aws/glue_dataset_client.py

```diff
@@ -9,8 +9,8 @@
 
 class DatasetCrawler:
     def __init__(self, dataset: Dataset):
-        session = SessionHelper.remote_session(accountid=dataset.AwsAccountId)
         region = dataset.region if dataset.region else 'eu-west-1'
```

Comment:
is dataset every nullable? do we need this line (L12)

Reply:
Maybe at some point it was, but region has been present in datasets since we open-sourced, so I am going to remove the if conditions. Good catch!